examples/website-demos/social-messaging/src/components/MessageInput/MessageInput.tsx

```diff
@@ -0,0 +1,161 @@
+import { useCallback, useEffect, useState } from 'react';
+import { ImageDropzone, FileUploadButton } from 'react-file-utils';
+
+import {
+  ChatAutoComplete,
+  EmojiPicker,
+  MessageInputProps,
+  UploadsPreview,
+  useMessageInputContext,
+  useChannelStateContext,
+} from 'stream-chat-react';
+
+import {
+  Attach,
+  CommandBolt,
+  EmojiPickerIcon,
+  GiphyIcon,
+  GiphySearch,
+  SendArrow,
+} from '../../assets';
+
+import { useGiphyContext } from '../../contexts/GiphyContext';
+
+import './MessageInput.scss';
```

Comment:
Can we rename the scss file to SocialMessageInput as well, to keep with the conventions of the rest of the project.